src/tests/ftest/osa/osa_offline_drain.py

```diff
@@ -61,25 +62,28 @@ def run_offline_drain_test(self, num_pool, data=False):
             pool[val].nvme_size.value = int(pool[val].nvme_size.value /
                                             num_pool)
             pool[val].create()
-            pool_uuid.append(pool[val].uuid)
             self.pool = pool[val]
+            self.pool.set_property("reclaim", "disabled")
+            test_seq = self.ior_test_sequence[0]
+
             if data:
-                self.run_ior_thread("Write", self.ior_dfs_oclass[0],
-                                    self.ior_apis[0],
-                                    self.ior_test_sequence[0])
+                self.run_ior_thread("Write", oclass, test_seq)
+                self.run_mdtest_thread()
 
-        # Drain the pool_uuid, rank and targets
+        # Drain rank and targets
         for val in range(0, num_pool):
             self.pool = pool[val]
             rank = rank + val
             self.pool.display_pool_daos_space("Pool space: Beginning")
             pver_begin = self.get_pool_version()
             self.log.info("Pool Version at the beginning %s", pver_begin)
+            if self.test_during_aggregation is True:
+                self.pool.set_property("reclaim", "time")
```

Comment:
the sleep is probably needed after enable reclaim, for waiting the batched DTXs commit.   Comments from niu
"right, wait for batched commit, but the 90 seconds was the old threshold, it was changed to 20 seconds long time ago"

I think this is probably only multiple replication object, instead of single replica obj.